### DISCUSSION TITLE
OOM crash when loading sample set after upgrading version

### DISCUSSION THREAD

**Discussion by ale64bit** [GrandOrgue/grandorgue#1373] [Help]
Today I updated to version 3.9.4-1 on Windows and it crashes when loading the [Begard](https://piotrgrabowski.pl/begard/) organ:

![image](https://user-images.githubusercontent.com/9343380/211087581-9cd33937-fe79-425f-8dbe-57e9ef30d2b0.png)

It used to work just fine before. I'm not sure where to get more logs to diagnose the problem but any suggestions would be welcome. The demo organ works fine.

**Comment by ale64bit** [comment]
It looks to be related to OOM, not to a specific version. Maybe some `malloc`s need to be checked somewhere when loading samples at startup.

**Comment by oleg68** [comment]
@ale64bit 
1. Is it a reproducible problem or it occured just once and not any more?
2. Could you attach the crash dump (usually it is created automatically in <User dir>\AppData\Local\CrashDumps. If none, then enable it https://helgeklein.com/blog/creating-an-application-crash-dump/)

**Comment by ale64bit** [comment]
1. It occurred consistently while I had a bunch of other apps open. As soon as I closed them all, it worked right away. However, when I tried to reproduce it now to gather the crash dump, it seems to not be happening anymore. I'm not sure if it has to do with having the sample cache already or some other factor.
2. I didn't have it enabled for GrandOrgue, but I do now. If it happens again, I'll send the crash dump.
